.circle/configure-rabbitmq.sh

```diff
@@ -2,7 +2,7 @@
 
 # Enable remote guest access
 CONFIG=$(cat <<EHD
-[{rabbit, [{disk_free_limit, 10}, {loopback_users, []}]}].
+[{rabbit, [{disk_free_limit, 10}, {loopback_users, []}, {tcp_listeners, [{"0.0.0.0", 5672}]}]}].
```

Comment:
Yeah :+1: 

Since there is once instance of `rabbitmq`.
And it's used by both packages (with docker) and your new unit + integration tests.